content/en/users/security/_index.md

```diff
@@ -2,24 +2,19 @@
 title: Security Services
 linkTitle: Security
 type: docs
-draft: true
-weight: 60
+weight: 70
 description: >
-  Security services in the EGI Cloud
+  Security services in the EGI infrastructure
 ---
 
 ## Using cloud services securely
 
 Security in the EGI Federated infrastructure is the highest priority.
-Scientists and researchers, who are the main users of the services offered by EGI,
-benefit from data centers and network architectures that are built to meet the
-requirements of the most security-sensitive institutions and organizations.
-
-The following services that can help you to configure and
-consume EGI services while meeting your security and compliance objectives:
+The following services can help you to configure and consume EGI services
+while meeting your security and compliance objectives:
 
 - The [Secrets Store](secrets-store) allows users to securely store credentials
-  or other sensitive information, and easily retrieve it when needed, instead of
-  hardcode it in code, configuration files, or environment variables.
+  or other sensitive information then easily retrieve it when needed, instead of
+  hardcoding it into code, configuration files, or environment variables.
```

Comment:
```suggestion
  , or other sensitive information, to later retrieve them instead of
  hardcoding them into code, configuration files, or environment variables.
```

Reply:
Disagree on the use of comma here (before "or"). Rephrased the rest.